packages/core/src/shipping/strategies/stripe-upe/stripe-upe-shipping-strategy.ts

```diff
@@ -33,11 +33,15 @@ export default class StripeUPEShippingStrategy implements ShippingStrategy {
     }
 
     async initialize(options: ShippingInitializeOptions): Promise<InternalCheckoutSelectors> {
-       if (!options.stripeupe) {
-           throw new InvalidArgumentError(`Unable to proceed because "options" argument is not provided.`);
-       }
+        if (!options.stripeupe) {
+            throw new InvalidArgumentError(`Unable to proceed because "options" argument is not provided.`);
+        }
+
+        if (!options.methodId) {
+            throw new InvalidArgumentError(`Unable to proceed because "options" argument is not provided.`);
+        }
 
-       const { container, gatewayId, methodId, onChangeShipping, getStyles, availableCountries, getStripeState } = options.stripeupe;
+        const { container, gatewayId, methodId, onChangeShipping, getStyles, availableCountries, getStripeState } = options.stripeupe;
```

Comment:
```suggestion
       const { container, gatewayId, methodId, onChangeShipping, getStyles, availableCountries, getStripeState } = options.stripeupe;
```

Reply:
This whole file's indentation is off, so I will leave it as is for now. Once we do the eslint update, this should be autofixed.